htdocs/public/members/new.php

```diff
@@ -539,6 +539,8 @@ function initmorphy()
 // Moral/Physic attribute
 $morphys["phy"] = $langs->trans("Physical");
 $morphys["mor"] = $langs->trans("Moral");
+print '<tr class="morphy"><td class="titlefield">'.$langs->trans('MemberNature').' <span style="color: red">*</span></td><td>'."\n";
```

Comment:
This is duplicate with following line...

Reply:
Good catch, I missed the deletion of the second line.
That line must appear both in the if and else, otherwise the table is broken.